### ISSUE TITLE
Trace ID link is using a wrong prefix 

### ISSUE THREAD

**Issue by simonswine** [grafana/tns#23]
On a log message with trace id within explore view, the linked trace is having an additional prefix `/grafana`, which leads to a 404:

```
http://localhost:8080/grafana/jaeger/trace/5cbf857baf072412
```
Correct would be (in my view) 

```
http://localhost:8080/jaeger/trace/5cbf857baf072412
```

**Comment by tomwilkie** [comment]
Is this a config option or a bug in Grafana? /cc @aocenas

**Comment by aocenas** [comment]
How is the link defined? Seems like the URL may have been defined as relative one.

**Comment by simonswine** [comment]
It is a relative URL in this deployment: https://github.com/grafana/tns/blob/46e681a70cc3a498c42c08d56715a12b6346c762/production/sample/default/main.jsonnet#L77

The above commit makes it work using `../`, althought the button label then contains '..'

**Comment by simonswine** [comment]
This is how it looks like with `/jaeger/trace/$${__value.raw}`:

![scrn-2020-07-13-15-41-08](https://user-images.githubusercontent.com/223048/87318237-f7c8a000-c51f-11ea-8def-682f3c008479.png)

And that with `../jaeger/trace/$${__value.raw}`:

![scrn-2020-07-13-15-13-55](https://user-images.githubusercontent.com/223048/87318241-f8f9cd00-c51f-11ea-95f1-d36b0edb6734.png)

**Comment by tomwilkie** [comment]
Oh this is all completely out of date - it should be a "datasource" link now: 

```
         jsonData: {
            derivedFields: [
              {
                matcherRegex: '(?:traceID|trace_id)=(\\w+)',
                name: 'TraceID',
                url: '$${__value.raw}',
                datasourceUid: 'ops-tools1-jaeger',
              },
            ],
          },
```

See https://github.com/grafana/deployment_tools/blob/master/ksonnet/environments/default/ops-tools1-us-east4.default/main.jsonnet#L257 - can you update this?

**Comment by simonswine** [comment]
Yes looking at it

**Comment by simonswine** [comment]
I am not too sure how to make the `datasourceUid` work. The database seems to contain a randomly generated one. I have tried to use `jaeger`, but not had much success. Where can I find the API docs for the specific jsonData in datasources? (https://grafana.com/docs/grafana/latest/http_api/data_source/ did not contain it)

**Comment by simonswine** [comment]
Ok I have found a way to set the `uid`, this should be fixed by #24
